lib/note-detail/index.tsx

```diff
@@ -249,6 +249,7 @@ export class NoteDetail extends Component<Props> {
     const {
       note,
       fontSize,
+      isSmallScreen,
```

Comment:
I don't think this is needed due to https://github.com/Automattic/simplenote-electron/pull/2037/files#diff-52e457c4f342fbeecf796739f2492ccaR385